static/docs/commands-reference/status.md

```diff
@@ -7,12 +7,12 @@ cache and remote cache.
 ## Synopsis
 
 ```usage
-usage: dvc status [-h] [-v] [-j JOBS] [--show-checksums] [-q] [-c]
-                  [-r REMOTE] [-a] [-T] [-d]
+usage: dvc status [-h] [-v] [-j JOBS] [--show-checksums]
+                  [-q] [-c] [-r REMOTE] [-a] [-T] [-d]
                   [targets [targets ...]]
 
 positional arguments:
-  targets               DVC files.
+  targets               Limit the scope to these stage files.
```

Comment:
update it to the new version?

Reply:
@shcheklein  Yes, updated it to newer version was waiting for original DVC repo to be updated. Since it got approved so pushed these changes now kindly review them

Reply:
@sanidhyamangal I think the DVC patch has been merged. Feel free to update the docs.